src/select/multiselect/ValueGroupRenderer.js

```diff
@@ -43,13 +45,14 @@ const ValueGroupRenderer = (props) => {
     }
     return null
   })
-  return <span>{values}</span>
+  return <React.Fragment>{values}</React.Fragment>
 }
 
 ValueGroupRenderer.propTypes = {
   allowSelectAll: PropTypes.bool,
   allOption: PropTypes.object,
   allSelectedText: PropTypes.string,
+  customDelimiter: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
```

Comment:
Instead of `customDelimiter`, suggest simplifying to `delimiter`.

Reply:
done!